Test/test2/Rlimitouts0.bpl.expect

```diff
@@ -0,0 +1,21 @@
+Prover error: line 105 column 10: max. resource limit exceeded
```

Comment:
The line/column number were from Z3 and we printed the error message out as it is. To make it less confusing, we will reprocess the "max. resource limit exceeded" so that it will report as "OutOfResource" instead of just an error.